test/e2e/pages/vl-accordion.page.js

```diff
@@ -20,18 +20,26 @@ class VlAccordionPage extends Page {
         return this._getAccordion('#accordion-javascript-toggle');
     }
 
-    async getJSAccordionOpenButton() {
-        return new VlButton(this.driver, '#open-accordion');
+    async clickJSAccordionOpenButton() {
+        return await this._clickJSAccordionButton('#open-accordion');
```

Comment:
Await is niet nodig:
```suggestion
        return this._clickJSAccordionButton('#open-accordion');
```